tuning/config/peft_config.py

```diff
@@ -79,4 +79,11 @@ class PromptTuningConfig:
     prompt_tuning_init: str = "TEXT"
     num_virtual_tokens: int = 8
     prompt_tuning_init_text: str = "Classify if the tweet is a complaint or not:"
-    tokenizer_name_or_path: str = "llama-7b-hf"
+    tokenizer_name_or_path: str = field(
```

Comment:
can this value be set automatically based on the value from the other tokenizer path?

Reply:
Yes, I would prefer if we can copy the value explicitly from ModelArguments, if that is set

Reply:
Just keeping the threads updated here as well.

This isn't possible since, copying the arg `tokenizer_name_or_path` explicitly from ModelArguments to PromptTuningConfig would not work since the current code unfolds it into kwargs to prepare the actual config object using `asdict` 
https://github.com/foundation-model-stack/fms-hf-tuning/blob/b655e1ae6f0e7cf7f93933c0cc2dd72023a8d0f3/tuning/utils/config_utils.py#L88
which stricly captures the arguments defined in the dataclass definition and does not capture those that are added over the object. This has been explained in Approach 2 of the PR description.

Reply:
hf_peft_config = PromptTuningConfig(
            task_type=task_type, **asdict(tuning_config), tokenizer_name_or_path=tokenizer_name_or_path
        )